src/.editorconfig

```diff
@@ -212,6 +212,27 @@ dotnet_diagnostic.CA2246.severity=silent
 dotnet_diagnostic.CA2255.severity=silent
 dotnet_diagnostic.CA2234.severity=silent
 dotnet_diagnostic.CA2252.severity=silent
+csharp_using_directive_placement = outside_namespace:silent
```

Comment:
>csharp_using_directive_placement [](http://example.com/codeflow?start=0&length=32)

This expansion of explicitly specified things resulted simply from me opening .editorconfig in VS to examine certain settings. As a result of these additions, dotnet format subsequently began to enforce the casing indentation I wanted.

So apparently, the literal absence of an explicit entry may be a clue to dotnet format skip certain reformattings.

Reply:
I was just about to offer your same rant. What I did, in fact, was go look at SPMI and some other projects for a pattern (not CASK). :)

Reply:
Turns out there's too many things in CASK to absorb quickly, so resolving this. Won't fix on first step forward. :)